ClassicsKotlin/app/src/main/java/com/android/tv/classics/receivers/TvLauncherReceiver.kt

```diff
@@ -104,20 +103,21 @@ class TvLauncherReceiver : BroadcastReceiver() {
      * Helper method used to retrieve a metadata item object given its program ID. Restricted API
      * is being used to compared programs based on their ID.
      */
-    @SuppressLint("RestrictedApi")
```

Comment:
This removal can be merged now that [bug #138150076](https://issuetracker.google.com/138150076) has been resolved.